assets/js/_js/default/contact_form.js

```diff
@@ -0,0 +1,17 @@
+function sendForm(alertText) {
+    var formInvalid = false;
+    var option = document.getElementById("select-form").value;
+    var name = document.getElementById("name").value;
+    document.getElementById("form-subject").value = option + "-" + name;
+    $('#contact-form input, #contact-form textarea ').each(function () {
+        if ($(this).val() === '') {
+            formInvalid = true;
+        }
+    });
+    if ($('#select-form :selected').val()=== ''){
+        formInvalid = true;
+    }
+    if (!formInvalid) {
+        alert(alertText);
```

Comment:
I can fill and send the form without seeing any alert messages.

Reply:
It happens when you don't fill the captcha, at localhost you can send the form without filling it, so the alert doesn't show.

Reply:
Then we cannot tested in local, it's not ideal. We'll see what happens after deployment then.